app/models/server/raw/Subscriptions.ts

```diff
@@ -25,7 +25,7 @@ export class SubscriptionsRaw extends BaseRaw<T> {
 		return this.find(query, options);
 	}
 
-	findByRoomIdAndNotUserId(roomId: string, userId: string, options: FindOneOptions<T> = {}): Cursor<T> {
+	findByRoomIdAndNotUserId(roomId: string, userId: string | undefined, options: FindOneOptions<T> = {}): Cursor<T> {
```

Comment:
I didn't understand why the second param is `undefined`. If the userID is not present, then I think we're only searching using `roomId`, so perhaps we should be using `findByRoomId` instead

Reply:
While I was in this file I saw this line as red underlined. I'll just try to revert back this line, push and see it it passes all the type check.

Reply:
@murtaza98 if I remove | undefined I get this warning where I call this function. Please refer the image below.
![image](https://user-images.githubusercontent.com/51796498/128549107-1cc0d53f-19b0-4b07-8e43-b6a68dac09a3.png)